packages/product-json-to-csv/src/map-product-data.js

```diff
@@ -402,8 +402,8 @@ export default class ProductMapping {
   _mapImagesToString(images: Array<Image>): string {
     return images
       .map((image: Image): string => {
-        const { url, label } = image
-        return label ? `${url}|${label}` : url
+        const { url, label, dimensions = { w: 0, h: 0 } } = image
+        return `${url}|${label || ''}|${dimensions.w}x${dimensions.h}`
```

Comment:
Does the platform set `w` and `h` to `0` by default? 

Otherwise, wouldn't it make sense to have the dimensions to also default to empty strings, like the label?

Reply:
When I create product from api playground  it is not allowing me to skip dimensions and even not w and h. So I thought both the things are mandatory so I have put it like this. It would be helpfull for the user to just enter number width and height not the separator which might they enter wrong. 

and yes in import they have default w and h reference here https://github.com/sphereio/sphere-node-product-csv-sync/blob/c5d2797131972189a05a0aa1764427ef56d90116/src/coffee/mapping.coffee#L468

Reply:
Great answer, thanks! 👍